tests/sqlparser_mysql.rs

```diff
@@ -222,6 +222,39 @@ fn parse_escaped_string() {
     }
 }
 
+#[test]
+fn parse_escaped_double_quoted_string() {
+    let sql = r#"SELECT "I\"m fine""#;
+
+    let stmt = mysql().one_statement_parses_to(sql, "");
+
+    match stmt {
+        Statement::Query(query) => match query.body {
+            SetExpr::Select(value) => {
+                let expr = expr_from_projection(only(&value.projection));
+                assert_eq!(
+                    *expr,
+                    Expr::Value(Value::DoubleQuotedString("I\"m fine".to_string()))
+                );
+            }
+            _ => unreachable!(),
+        },
+        _ => unreachable!(),
+    };
```

Comment:
You could potentially write this in a single match statement like


```suggestion
    match stmt {
        Statement::Query({ 
          body: SetExpr::Select(value),
          ..
        } => {
                let expr = expr_from_projection(only(&value.projection));
                assert_eq!(
                    *expr,
                    Expr::Value(Value::DoubleQuotedString("I\"m fine".to_string()))
                );
        },
        _ => unreachable!(),
    };
```